common/src/main/java/com/wynntils/core/config/ConfigManager.java

```diff
@@ -183,7 +184,7 @@ private static void loadFeatureOverlayConfigOptions(FeatureCategory category, Fe
         }
     }
 
-    private static List<ConfigHolder> getConfigOptions(FeatureCategory category, Configurable parent) {
+    private static List<ConfigHolder> getConfigOptions(FeatureCategory category, AbstractConfigurable parent) {
```

Comment:
```suggestion
    private static List<ConfigHolder> getConfigOptions(FeatureCategory category, Configurable parent) {
```

The `AbstractConfigurable` class should like only be used when declaring the superclass of the configurable subclasses. Elsewhere, the interface should be used.